apps/ledger-live-mobile/src/const/navigation.ts

```diff
@@ -397,6 +398,7 @@ export enum ScreenName {
   OnboardingImportSelectAccount = "OnboardingImportSelectAccount",
   OnboardingQuiz = "OnboardingQuiz",
   OnboardingQuizFinal = "OnboardingQuizFinal",
+  OnboardingBlePairing = "OnboardingBlePairing",
```

Comment:
Can it be named `OnboardingBleDevicePairingFlow` pls ? It's a "flow" because it's not only the pairing step but also the device scanning step